<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>J-MEDIC Registro de Médicos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background-color: rgb(0, 0, 0);
    font-family: Arial;
    color: rgb(255, 255, 255);
}

QMenuBar{
    text-decoration: underline rgb(255, 255, 255);
}

QLabel{
    font: 75 18pt;
}

QLineEdit,QComboBox,QDoubleSpinBox,QSpinBox{
    color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
    font: 75 14pt &quot;Arial&quot;;
}

QPushButton{
  color: #fff !important;
  font-size: 20px;
  font-weight: 500;
  background: #318aac;
  border: 2px solid;
  border-color: #318aac;
  position: relative;
}

QPushButton:before {
  content:&quot;&quot;;
  position: absolute;
  top: 0px;
  left: 0px;
  width: 100%;
  height: 0px;
  background: rgba(255, 255, 255, 0.1);
  transition: all 1s ease;
}

QPushButton:hover:before {
  height: 100%;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Registar Médicos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nombre">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Nombre&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ApellidoP">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Apellido Paterno&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ApellidoM">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Apellido Materno&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Cedula">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cedula&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Telefono">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Teléfono&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_Turno_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matutino</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vespertino</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Turno">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Turno&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Nombre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ApellidoP">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ApellidoM">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Cedula">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Telefono">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_guardar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Usuario&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_user">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Constraseña&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_contra">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_turno">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <zorder>input_turno</zorder>
   <zorder>label</zorder>
   <zorder>label_Nombre</zorder>
   <zorder>label_ApellidoP</zorder>
   <zorder>label_ApellidoM</zorder>
   <zorder>label_Cedula</zorder>
   <zorder>label_Telefono</zorder>
   <zorder>input_Turno_2</zorder>
   <zorder>label_Turno</zorder>
   <zorder>input_Nombre</zorder>
   <zorder>input_ApellidoP</zorder>
   <zorder>input_ApellidoM</zorder>
   <zorder>input_Cedula</zorder>
   <zorder>input_Telefono</zorder>
   <zorder>boton_guardar</zorder>
   <zorder>label_2</zorder>
   <zorder>input_user</zorder>
   <zorder>label_3</zorder>
   <zorder>input_contra</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRegresar"/>
    <addaction name="actionLista_de_M_dicos"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
  <action name="actionVer">
   <property name="text">
    <string>Ver</string>
   </property>
  </action>
  <action name="actionLista_de_M_dicos">
   <property name="text">
    <string>Lista de Médicos</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../image/guardar.qrc"/>
 </resources>
 <connections/>
</ui>
